packages/patternfly-4/react-docs/src/components/example/liveDemo.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from './example.styles';
+import PropTypes from 'prop-types';
+import * as CoreComponents from '@patternfly/react-core';
+import * as CoreIcons from '@patternfly/react-icons';
+import { LiveProvider, LiveEditor, LiveError, LivePreview, withLive } from 'react-live';
+import { transform } from 'babel-standalone';
+
+const propTypes = {
+  className: PropTypes.string,
+  raw: PropTypes.string.isRequired,
+  images: PropTypes.array
+};
+
+const defaultProps = {
+  className: '',
+  images: []
+};
+
+const scopePlayground = { React, ...CoreComponents, ...CoreIcons };
+
+const transformCode = code => {
+  try {
+    // LiveEditor doesn't work properly with these so need to remove
+    code = code.replace(/^\s*import.*$/gm, '');
```

Comment:
Are you able to do this on gatsby node? It might speed up client-side rendering. Not a big deal if it is not possible though.

Reply:
Good suggestion, I will try it out

Reply:
Had trouble getting the raw file contents in gatsby-node. Going to stick with this for now.